index.html

```diff
@@ -0,0 +1,264 @@
+      </section>
+    </header>
+    <!-- MAIN SECTION -->
+    <main class="main-container">
+      <section class="main-program">
+        <div class="main-program-header">
+          <h2>Main Program</h2>
+          <div class="underline"></div>
+        </div>
+        <div class="card-container">
+          <div class="card">
+            <div class="card-header">
+              <div class="card-header-img">
+                <img
+                  src="./assets/Images/lecture-icon-black.PNG"
+                  alt="purchase-icon"
+                />
+              </div>
+            </div>
+            <div class="card-body">
+              <h3>Lecture</h3>
+              <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-header">
+              <div class="card-header-img">
+                <img
+                  src="./assets/Images/exhibition-icon-black.PNG"
+                  alt="Exhibition-icon"
+                />
+              </div>
+            </div>
+            <div class="card-body">
+              <h3>Exhibitions</h3>
+              <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-header">
+              <div class="card-header-img">
+                <img
+                  src="./assets/Images/forum-icon-black.PNG"
+                  alt="forum-icon"
+                />
+              </div>
+            </div>
+            <div class="card-body">
+              <h3>Forum</h3>
+              <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-header">
+              <div class="card-header-img">
+                <img
+                  src="./assets/Images/workshop-icon-black.PNG"
+                  alt="workshop-icon"
+                />
+              </div>
+            </div>
+            <div class="card-body">
+              <h3>Workshop</h3>
+              <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-header">
+              <div class="card-header-img">
+                <img
+                  src="./assets/Images/ignite-icon-black.PNG"
+                  alt="ignite-icon"
+                />
+              </div>
+            </div>
+            <div class="card-body">
+              <h3>CF Ignite</h3>
+              <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+            </div>
+          </div>
+        </div>
+        <button class="btn-conference">Join Conference Fora 2022</button>
+        <div class="card-bottom-txt">
+          <h3>see the whole program</h3>
+          <div class="underline"></div>
+        </div>
+      </section>
```

Comment:
- [ ] Your 'Main Program' section does not have the design of the mobile template on screens with a width between 480px and 768px and does not have the design of the desktop template on screens between 768px and 1200px. Kindly make this section vertical on the mobile view(up to 768px) with one 'program' per line and horizontal(above 768px) with one line of 5 columns of 'programs' like the template.
 -  Your design:
 - Between 480 and 768:
 - 
<img width="659" alt="image" src="https://user-images.githubusercontent.com/61919617/203632133-64cb3aee-934e-48e5-ba39-7bf0ebe81b64.png">

 - Between 768 and 1200:
 - 
<img width="982" alt="image" src="https://user-images.githubusercontent.com/61919617/203632193-423ea6bc-9fdd-4644-b4ab-c8299a9414d6.png">